components/brave_new_tab_ui/stories/regular.tsx

```diff
@@ -49,6 +50,13 @@ export default {
   ]
 }
 
+updateImages(images.map((image): NewTab.BraveBackground => {
+  return {
+    ...image,
+    wallpaperImageUrl: require('../../img/newtab/backgrounds/' + image.wallpaperImageUrl)
```

Comment:
Just double checking `require` is kosher in stories :smile:

Reply:
Honestly, I copied this from existing code. I traced back where it started. It seems that it's been used quite a long time. https://github.com/brave/brave-core/pull/4470/files#diff-d2a9ef04780c18cbe6031be2e1e963e7b94ea5dd6429e4028b75ec417e6b7de8

Due to my lack of expertise in js, I'm not sure what I can do instead of this. Do you have a suggestion on this? I'm all ears :)

Reply:
Ah, if it was already used I'm sure it is - require is the nodeJS way of importing things, from before we had es imports. I'm hazy on exactly what the difference is :laughing: